src/libecalc/presentation/flow_diagram/fde_models.py

```diff
@@ -76,4 +76,4 @@ class Node(BaseModel):
     model_config = ConfigDict(frozen=True)
 
 
-FlowDiagram.update_forward_refs()
+FlowDiagram.model_rebuild()
```

Comment:
I think this can be removed instead of replaced.

Reply:
Why is that? I dont understand.
https://docs.pydantic.dev/latest/concepts/models/#rebuilding-model-schema 
_Pydantic tries to determine when this is necessary automatically and error if it wasn't done, but you may want to call [model_rebuild()](https://docs.pydantic.dev/latest/api/base_model/#pydantic.BaseModel.model_rebuild) proactively when dealing with recursive models or generics._

Reply:
Just because it seemed to work without it. I didn't check the actual schema. I'm guessing it's fine since all the classes will be defined when using the classes.

But seeing as they mention calling it proactively we might as well do that.